espaloma/mm/tests/test_openmm_consistency.py

```diff
@@ -202,7 +202,7 @@ def test_energy_angle_and_bond(g):
 
     # print(g.nodes['n2'].data)
     esp.mm.geometry.geometry_in_graph(g.heterograph)
-    esp.mm.energy.energy_in_graph(g.heterograph, terms=["n2", "n3", "n4", "n4_improper"])
+    esp.mm.energy.energy_in_graph(g.heterograph, terms=["n2", "n3", "n4"])
```

Comment:
Should we somehow add a note that the `n4_improper` is not implemented correctly, and that we will need to add test coverage later?